<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWinTabBoxConfigForm</class>
 <widget class="QWidget" name="KWinTabBoxConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>305</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KTitleWidget" name="description">
     <property name="text">
      <string>These settings are used by the &quot;Walk Through Windows&quot; actions.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>List windows:</string>
       </property>
       <property name="buddy">
        <cstring>listModeCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KComboBox" name="listModeCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Current Desktop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>All Desktops</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Current Desktop Grouped by Applications</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>All Desktops Grouped by Applications</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Sort order:</string>
       </property>
       <property name="buddy">
        <cstring>switchingModeCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KComboBox" name="switchingModeCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Recently used</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stacking order</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Minimized:</string>
       </property>
       <property name="buddy">
        <cstring>minimizedModeCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KComboBox" name="minimizedModeCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Ignore status</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Exclude minimized windows</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Minimized windows only</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="showDesktopBox">
     <property name="toolTip">
      <string>Adds an entry to minimize all windows.</string>
     </property>
     <property name="text">
      <string>Include desktop</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="highlightWindowCheck">
     <property name="toolTip">
      <string>The currently selected window will be highlighted by fading out all other windows. This option requires desktop effects to be active.</string>
     </property>
     <property name="text">
      <string>Highlight selected window</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showOutlineCheck">
     <property name="text">
      <string>Show outline of selected window</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Effect:</string>
       </property>
       <property name="buddy">
        <cstring>effectCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="KComboBox" name="effectCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>The effect to replace the list window when desktop effects are active.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="effectConfigButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="effectInfoButton"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,2">
     <item>
      <widget class="QCheckBox" name="showTabBox">
       <property name="text">
        <string>Display list while switching</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listModeCombo</tabstop>
  <tabstop>switchingModeCombo</tabstop>
  <tabstop>showOutlineCheck</tabstop>
  <tabstop>effectCombo</tabstop>
  <tabstop>effectConfigButton</tabstop>
  <tabstop>effectInfoButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>showTabBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>layoutConfigButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
